ros/processor/event_producer.py

```diff
@@ -0,0 +1,76 @@
+import json
+from flask import request
+from ros.lib import produce
+from confluent_kafka import KafkaError
+from datetime import datetime, timezone
+from ros.lib.models import PerformanceProfile
+from ros.lib.config import NOTIFICATIONS_TOPIC, get_logger
+from ros.lib.utils import org_id_from_identity_header, systems_ids_for_existing_profiles
+
+logger = get_logger(__name__)
+
+
+# Event for new suggestion
+def new_suggestion_event(host, platform_metadata, previous_state, current_state):
+
+    org_id = org_id_from_identity_header(request)
+    query = systems_ids_for_existing_profiles(org_id)
+    systems_with_suggestions = query.filter(PerformanceProfile.number_of_recommendations > 0).count()
+    request_id = platform_metadata.get('request_id')
+    payload = {
+        "version": "v1.0.0",
+        "bundle": "rhel",
+        "application": "ros",
+        "event_type": "new-suggestion",
+        "timestamp": datetime.now(timezone.utc).isoformat(),
+        "account_id": host.get("account_id") or "",
+        "org_id": host.get("org_id"),
+        "context": {
+            "event_name": "New suggestion",
+            "systems_with_suggestions": systems_with_suggestions
+        },
+        "rh-message-id": platform_metadata.get("request_id"),
```

Comment:
This line will cause a notification schema validation issue if you test the payload with [our validator](https://internal.console.stage.redhat.com/api/notifications/internal/index.html#/utils/message-validator):
```
$.rh-message-id: is not defined in the schema and the schema does not allow additional properties
```
The `rh-message-id` field is expected as a Kafka header but it's prefectly fine to omit it at first and add it later. If that header is present, your Kafka messages will be [protected against duplicate processing](https://core-platform-apps.pages.redhat.com/notifications-docs/dev/user-guide/send-notification.html#_protecting_your_kafka_messages_against_duplicate_processing).